junit-jupiter-params/src/test/java/org/junit/jupiter/params/provider/MockCsvAnnotationBuilder.java

```diff
@@ -23,6 +23,9 @@
 	static CsvSource csvSource(String... lines) {
 		return csvSource().lines(lines).build();
 	}
+	static CsvSource csvSource(boolean ignoreTrailingAndLeadingWhitespace, String... lines) {
```

Comment:
The introduction of this method makes the tests less readable, please inline `svSource().lines(lines).ignoreTrailingAndLeadingWhitespace(ignoreTrailingAndLeadingWhitespace).build()` in your test methods.

Reply:
ok

Reply:
done, please check the latest commit.

Reply:
This isn't done, yet.